metadata-load/src/main/java/uk/ac/ebi/ampt2d/metadata/importer/configuration/MetadataImporterMainApplicationConfiguration.java

```diff
@@ -66,7 +67,7 @@ public ReferenceSequenceXmlRetrieverThroughEntrezApi assemblyXmlRetrieverThrough
 
     @Bean
     @ConditionalOnProperty(name = "import.source", havingValue = "API")
-    public ObjectsImporter objectImporterThroughEnaApi(SraXmlRetrieverThroughApi sraXmlRetrieverThroughApi,
+    public ObjectsImporter objectImporterThroughEnaApi(SraXmlRetrieverByAccession sraXmlRetrieverByAccession,
```

Comment:
Actually I think that here you need to make it `SraXmlRetrieverByAccession sraXmlRetrieverThroughApi`. So you depend on an interface (`SraXmlRetrieverByAccession`), but name a variable after a specific class (`sraXmlRetrieverThroughApi`). This way, Spring will know which specific bean to inject. Currently it highlights as a multiple autowire error:
![a](https://user-images.githubusercontent.com/10669118/68589988-4d2a4a00-0485-11ea-81bd-5dc9f014bf71.png)